package.json

```diff
@@ -5,6 +5,7 @@
   "main": "dist/createjs-accessibility.js",
   "scripts": {
     "build": "rm -rf dist && NODE_ENV=production webpack",
+    "lint": "eslint --fix --color src",
```

Comment:
I would expect `npm run lint` to report linting issues, not try to fix them.

Reply:
Done